src/trezor.ts

```diff
@@ -708,19 +660,23 @@ export class TrezorExportExtendedPublicKey extends TrezorExportHDNode {
  * const signature = await interaction.run();
  * console.log(signatures);
  * // ["ababab...", // 1 per input]
- * @extends {module:trezor.TrezorInteraction}
  */
 export class TrezorSignMultisigTransaction extends TrezorInteraction {
-  /**
-   * @param {object} options - options argument
-   * @param {string} options.network - bitcoin network
-   * @param {UTXO[]} [options.inputs] - inputs for the transaction
-   * @param {TransactionOutput[]} [options.outputs] - outputs for the transaction
-   * @param {string[]} [options.bip32Paths] - BIP32 paths on this device to sign with, one per each input
-   * @param {string} [options.psbt] - PSBT string encoded in base64
-   * @param {object} [options.keyDetails] - Signing Key Details (Fingerprint + bip32 prefix)
-   * @param {boolean} [options.returnSignatureArray] - return an array of signatures instead of a signed PSBT (useful for test suite)
-   */
+  // network: Network
+  inputs: any[];
+
+  outputs: any[];
+
+  bip32Paths: string[];
+
+  psbt?: string;
+
+  keyDetails: any;
```

Comment:
I think this can use WalletConfigKeyDerivation for this?

Reply:
I think this can be removed.

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff